tests/container_upgrade/container_upgrade_helper.py

```diff
@@ -28,6 +28,11 @@
     "docker-auditd-watchdog": "auditd_watchdog",
 }
 
+service_list = [
```

Comment:
Maybe better to rename like existing_service_list? Auditd is also one of the services we test but wouldn't need the logic to stop, tag, restart.